src/attest.ts

```diff
@@ -22,27 +21,22 @@ export const createAttestation = async (
 ): Promise<AttestResult> => {
   // Sign provenance w/ Sigstore
   const attestation = await attest({
-    subjectName: subject.name,
-    subjectDigest: subject.digest,
+    subjects,
     predicateType: predicate.type,
     predicate: predicate.params,
     sigstore: opts.sigstoreInstance,
     token: opts.githubToken
   })
 
-  const subDigest = subjectDigest(subject)
-  const result: AttestResult = {
-    ...attestation,
-    subjectName: subject.name,
-    subjectDigest: subDigest
-  }
+  const result: AttestResult = attestation
 
-  if (opts.pushToRegistry) {
+  if (subjects.length === 1 && opts.pushToRegistry) {
```

Comment:
This means that we don't push multi subject attestations to an OCI registry, right? If so then it should be part of the updated readme I think.

Reply:
There's really no change in behavior here. Currently, the only way to identify multiple subjects when calling the action is through a file glob. When identifying an image by name & digest you are limited to a single subject.

There have been a number of requests to support the attestation of multiple images at the same time, but we haven't yet tackled this work.